modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -54,10 +54,27 @@ public class UpgradeCatchAllCheck {
 			companyId, 551, "test", start, end);
 	}
 
+	public void method(
+			String externalReferenceCode, long cProductId,
+			String cpInstanceUuid, long commercePriceListId, BigDecimal price,
+			BigDecimal promoPrice, ServiceContext serviceContext)
+		throws PortalException {
+
+		_commercePriceEntryLocalService.addCommercePriceEntry(
+			externalReferenceCode, cProductId, cpInstanceUuid,
+			commercePriceListId, price, promoPrice, serviceContext);
+
+		_commercePriceEntryLocalService.addCommercePriceEntry(
+		    null, cpInstance.getCPDefinition().getCProductId(), cpInstance.getCPInstanceUuid(),
+		    commercePriceList.getCommercePriceListId(), price, null, serviceContext);
+	}
+
 	@Reference
 	private AssetVocabularyLocalService _assetVocabularyLocalService;
 
 	@Reference
 	private AssetVocabularyService _assetVocabularyService;
 
+	private CommercePriceEntryLocalService _commercePriceEntryLocalService;
+
```

Comment:
@alberttgomes, please follow the pattern, all new references must have the `@Reference` notation